litellm/model_prices_and_context_window_backup.json

```diff
@@ -970,6 +970,17 @@
         "supports_function_calling": true,
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "gemini-1.5-pro-preview-0409": {
+        "max_tokens": 8192,
+        "max_input_tokens": 1000000,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0,
```

Comment:
why is cost 0 ?

Reply:
we use the openai conversion of 1 token ~= 4 chars